.github/workflows/development.yml

```diff
@@ -0,0 +1,33 @@
+name: CI/CD Pipeline
```

Comment:
I think we can change the name of the file because you only have one environment, and most of the time, you do the same for every environment